docs/server/source/appendices/run-with-docker.md

```diff
@@ -30,6 +30,17 @@ docker run \
   -y configure \
   [mongodb|rethinkdb]
 
+# For Docker for Mac users
+docker run \
+  --interactive \
+  --rm \
+  --tty \
+  --volume $HOME/bigchaindb_docker:/data \
+  --env BIGCHAINDB_DATABASE_HOST=172.17.0.1 \
```

Comment:
why do we need a hard coded value here?

Reply:
Because it is the default docker bridge gateway.

Reply:
Maybe add a comment about that in the text, otherwise it seems rather arbitrary.